hs_core/templates/pages/my-resources.html

```diff
@@ -262,11 +262,13 @@ <h4 class="panel-title">
                             {# Last Modified #}
                             <td>{{ res.last_updated|date:"M d, Y" }} at {{ res.last_updated|time }}</td>
                             <td>
+                                {# Here we are hitting the database once for each resource that is listed #}
```

Comment:
I wonder if we could use prefetch to speed these loops up.

Reply:
Since 'metadata' is a property and not a model field, won't be possible to prefetch.

Reply:
Maybe we can remove this abstraction once we've moved away from other resource types.

Reply:
Now prefetching `content_object` (metadata).